config.py

```diff
@@ -72,6 +72,7 @@ class Config(object):
     ACCOUNT_EMAIL_CHANGE_ENABLED = bool(strtobool(os.getenv('ACCOUNT_EMAIL_CHANGE_ENABLED', "False")))
     SHARE_SURVEY_ENABLED = bool(strtobool(os.getenv('SHARE_SURVEY_ENABLED', "True")))
     MAX_SHARED_SURVEY = int(os.getenv('MAX_SHARED_SURVEY', '50'))
+    TECHNICAL_MESSAGE_ENABLED = bool(strtobool(os.getenv('SHARE_SURVEY_ENABLED', "True")))
```

Comment:
```suggestion
    TECHNICAL_MESSAGE_ENABLED = bool(strtobool(os.getenv('TECHNICAL_MESSAGE_ENABLED', "True")))
```

Reply:
Thanks, it would have been a disaster 👍 , will change it

Reply:
fixed